app/Console/Commands/UpdateOrchids.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use App\Profile;
+use App\ProfileData;
+
+
+class UpdateOrchids extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'orchid:update';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Loop through the profiles with orcid managed publications to sync the info';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $inc = 0;
+        $profiles = Profile::whereHas('data', function($query) { $query->where('type', 'information')->where('data->orc_id_managed', '1'); } )->get();
```

Comment:
1. Style: this should probably be written on multiple lines, like the example in the documentation: https://laravel.com/docs/8.x/eloquent-relationships#querying-relationship-existence
2. The subquery should probably also include a `->whereNotNull('data->orc_id')` clause to make sure that the profile actually has an ORCiD to look up.